lib/logstash/plugin_mixins/rabbitmq_connection.rb

```diff
@@ -91,6 +94,9 @@ def setup_rabbitmq_connection_config
         # Extra queue arguments as an array.
         # To make a RabbitMQ queue mirrored, use: `{"x-ha-policy" => "all"}`
         config :arguments, :validate => :array, :default => {}
+
+        # Share a connection with another plugin whose pool name and paramerters are identical (host, port, vhost and username)
```

Comment:
Hmmm, this could use a bit more of an explanation, but I may just add that in a subsequent commit.